content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -164,10 +163,11 @@ committing unwanted files into Git (e.g. once successful experiments are
 
 </details>
 
-💡 To clear the experiments queue and start over, use `dvc exp remove --queue`.
+💡 To clear the experiments queue and start over, use
+`dvc queue remove --queued`.
```

Comment:
Can we include this change in content/docs/user-guide/experiment-management/cleaning-experiments.md too please? It's the only other guide where it's mentioned.

Reply:
Note: I open https://github.com/iterative/dvc.org/pull/3894 to address this and the remaining stuff I mentioned here.